README.md

```diff
@@ -205,17 +183,11 @@ Example:
 
 > Mention all of the collaborators of this project.
```

Comment:
Please remove this line because it's only an instruction of what you can put here

Reply:
- [ ] like the previous reviewer noted, kindly remove the comments on the README, as they are just there to help direct you to the information to add in the various section. Removing them will keep your README clean and professional. Nice work